src/main/java/org/broadinstitute/dsm/model/elasticsearch/ESAddress.java

```diff
@@ -61,4 +64,6 @@ public String getPhone() {
     public String getRecipient() {
         return recipient;
     }
+
+    public boolean getValid() { return valid; }
```

Comment:
usually I prefer to use `is/has` prefixes for boolean getters, I think it makes more understandable.